1.generate-profiles/1.aggregate.py

```diff
@@ -0,0 +1,92 @@
+import os
+import sys
+import pathlib
+import argparse
+import warnings
+import pandas as pd
+
+from pycytominer import aggregate
+from pycytominer.cyto_utils import output
+
+sys.path.append("../scripts")
+from config_utils import process_config_file
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--config_file",
+    help="configuration yaml file for the profiling pipeline",
+    default="profiling_config.yaml",
+)
+args = parser.parse_args()
+config_file = args.config_file
+
+config = process_config_file(config_file)
+
+# Extract config arguments
+core_args = config["core"]
+batch = core_args["batch"]
+trash_files = core_args["trash_files"]
```

Comment:
Per our recent conversation about the site_processing_config, trash_files should become ignore_files